.github/workflows/AutomaticRelease.yml

```diff
@@ -0,0 +1,280 @@
+name: Automatic release
+
+on:
+  workflow_dispatch:
+    inputs:
+      update_timber_api:
+        type: boolean
+        description: Update TimberAPI
+        default: false
+        required: true
+      timber_api_update_type:
+        type: choice
+        default: beta
+        description: Update type
+        required: false
+        options:
+          - major
+          - feature
+          - bug
+          - alpha
+          - beta
+          - rc
+      update_timber_api_example:
+        type: boolean
+        description: Update TimberAPIExample
+        required: true
+      timber_api_example_update_type:
+        type: choice
+        default: beta
+        description: Update type
+        required: false
+        options:
+          - major
+          - feature
+          - bug
+          - alpha
+          - beta
+          - rc
+
+
+env:
+  #Timber API
+  timber_api_release_name: "TimberAPI"
+  timber_api_project_path: "TimberAPI/TimberAPI.csproj"
+  timber_api_output_path: "TimberAPI/bin/RELEASE/netstandard2.1"
+  timber_api_package_dir: "TimberAPI/ThunderstorePackage"
+  timber_api_nuget_path: "TimberAPI/bin/RELEASE"
+  #Timber API Example
+  timber_api_example_release_name: "TimberAPI Example"
+  timber_api_example_project_path: "TimberAPIExample/TimberAPIExample.csproj"
+  timber_api_example_output_path: "TimberAPIExample/bin/RELEASE/netstandard2.1"
+  timber_api_example_package_dir: "TimberAPIExample/ThunderstorePackage"
+
+
+jobs:
+  # Automatic updating semantic versioning
+  versions:
+    name: Parse project versions
+    runs-on: ubuntu-latest
+    outputs:
+      timber_api: ${{ github.event.inputs.update_timber_api == 'false' && steps.timber_api.outputs.match || steps.bump_timber_api_version.outputs.next-version }}
+      timber_api_example: ${{ github.event.inputs.update_timber_api_example == 'false' && steps.timber_api_example.outputs.match || steps.bump_timber_api_example_version.outputs.next-version }}
+    steps:
+      - uses: actions/checkout@v3
+
+      # Loading versioning file
+      - name: Read versions.txt
+        id: read_versions_file
+        uses: juliangruber/read-file-action@v1
+        with:
+          path: ./project_versions.txt
+
+      # TimberAPI versioning
+      - id: timber_api
+        uses: actions-ecosystem/action-regex-match@v2
+        with:
+          text: ${{ steps.read_versions_file.outputs.content }}
+          regex: '(?<=TIMBER_API=).*'
+
+      - id: bump_timber_api_version
+        uses: christian-draeger/increment-semantic-version@1.0.2
```

Comment:
Neat tool, but I think you too pointed out that bumping version to alpha, beta or rc with this it doesnt bump other version number with it, which makes releasing betas a little hard. This is good for now, but maybe even forking this tool and modifying it to our needs in the future is possible?